README.md

```diff
@@ -200,16 +200,31 @@ These options may also be configured from knife.rb, as in this example:
     :kerberos_service             The Kerberos service used for authentication
     :ca_trust_file                The Certificate Authority (CA) trust file used for SSL transport
 
+#### Options to configure WinRM for Bootstrapping a Windows Node
+
+    :winrm_max_timeout           Set winrm max timeout in miliseconds
```

Comment:
We should probably mention what these are useful for, i.e. if you have long-running run-lists and if the chef run might use a lot of memory. We should also state that in most cases people don't need to set these, but if they see certain error codes (e.g. 500?) during bootstrap, particularly on Win2k8r2, it may make sense to move these beyond the default.